OneBlink.SDK/models/FormSubmissionEvent.cs

```diff
@@ -188,18 +188,46 @@ public static FormSubmissionEvent CreateWestpacQuickWebSubmissionEvent(Guid elem
             fseconfig.excludedElementIds = excludedElementIds;
             fseconfig.usePagesAsBreaks = usePagesAsBreaks;
             fseconfig.emailTemplate = emailTemplate;
-            FormSubmissionEvent pdfEvent = new FormSubmissionEvent();
-            pdfEvent.type = "PDF";
-            pdfEvent.configuration = fseconfig;
-            pdfEvent.isDraft = isDraft;
-            pdfEvent.conditionallyExecute = conditionallyExecute;
-            pdfEvent.requiresAllConditionallyExecutePredicates = requiresAllConditionallyExecutePredicates;
+            FormSubmissionEvent emailEvent = new FormSubmissionEvent();
+            emailEvent.type = "PDF";
+            emailEvent.configuration = fseconfig;
+            emailEvent.isDraft = isDraft;
+            emailEvent.conditionallyExecute = conditionallyExecute;
+            emailEvent.requiresAllConditionallyExecutePredicates = requiresAllConditionallyExecutePredicates;
             if (conditionallyExecutePredicates != default(List<ConditionallyShowPredicate>))
             {
-                pdfEvent.conditionallyExecutePredicates = conditionallyExecutePredicates;
+                emailEvent.conditionallyExecutePredicates = conditionallyExecutePredicates;

```

Comment:
Don't think all these should be renamed

Reply:
Whoops, my bad, I did change all occurrences when I really should not have